lib/commands/run.ts

```diff
@@ -116,8 +116,10 @@ export class RunCommandBase implements ICommand {
 			watchAllFiles: this.$options.syncAllFiles,
 			clean: this.$options.clean
 		};
-
-		await this.$liveSyncService.liveSync(deviceDescriptors, liveSyncInfo);
+		const projectFilesConfig: IProjectFilesConfig = {
```

Comment:
these lines are introduced on a lot of places, maybe you can introduce a method in [helpers](https://github.com/telerik/mobile-cli-lib/blob/master/helpers.ts)